pkg/cmd/datamgr/cli/server/server.go

```diff
@@ -287,7 +283,6 @@ func newServer(f client.Factory, config serverConfig, logger *logrus.Logger) (*s
 		namespace:             f.Namespace(),
 		metricsAddress:        config.metricsAddress,
 		kubeClient:            kubeClient,
-		veleroClient:          veleroClient,
```

Comment:
no users of veleroClient?

Reply:
Yes, veleroClient is unused in data-manager. And it should not be used.